zcncore/wallet.go

```diff
@@ -340,12 +361,23 @@ func WithConfirmationChainLength(m int) func(c *ChainConfig) error {
 }
 
 // InitZCNSDK initializes the SDK with miner, sharder and signature scheme provided.
-func InitZCNSDK(blockWorker string, signscheme string, configs ...func(*ChainConfig) error) error {
-	if signscheme != "ed25519" && signscheme != "bls0chain" {
+func InitZCNSDK(cfg *conf.Config) error {
```

Comment:
@cnlangzi I think InitZCNSDK is used in wasm 0proxy and it's very unlikely to use pointers and data structures in function params, very often they can't be converted to mobiles/wasm library. Consider leaving the same params as before and construct conf.Config inside function

Reply:
+1 to the comment. can we also pass the config in json or string instead of pointer?